demo_nodes_cpp/src/parameters/parameter_events_async.cpp

```diff
@@ -63,9 +57,15 @@ class ParameterEventsAsyncNode : public rclcpp::Node
     // Setup callback for changes to parameters.
     parameter_event_sub_ = parameters_client_->on_parameter_event(on_parameter_event_callback);
 
+    // Declare parameters that may be set on this node
+    this->declare_parameter("foo");
+    this->declare_parameter("bar");
+    this->declare_parameter("baz");
+    this->declare_parameter("foobar");
+
     // Queue a `set_parameters` request as soon as `spin` is called on this node.
     // TODO(dhood): consider adding a "call soon" notion to Node to not require a timer for this.
-    timer_ = create_wall_timer(0s,
+    timer_ = create_wall_timer(200ms,
```

Comment:
What's the purpose of this change?

Reply:
It's the equivalent of the `rclcpp::sleep_for` above, same issue with parameter events demos. Though the actual delay is completely empirical and even arbitrarly, I give you that. See https://github.com/ros2/demos/issues/328 for the tracking issue.